src/main/java/org/json/JSONObject.java

```diff
@@ -215,14 +214,13 @@ public JSONObject(JSONTokener x) throws JSONException {
         if (x.nextClean() != '{') {
             throw x.syntaxError("A JSONObject text must begin with '{'");
         }
+        c = x.nextClean();
```

Comment:
Please revert all changes in this method.

Reply:
done, all changes in this method are reverted.